dev/fattest.simplicity/src/com/ibm/ws/fat/util/tck/TestLoggingObserver.java

```diff
@@ -0,0 +1,26 @@
+/*******************************************************************************
+ * Copyright (c) 2018 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.fat.util.tck;
+
+import org.jboss.arquillian.core.api.annotation.Observes;
+import org.jboss.arquillian.core.spi.EventContext;
+import org.jboss.arquillian.test.spi.event.suite.TestEvent;
+
+public class TestLoggingObserver {  
+      
+    void logTestStartStop(@Observes(precedence = 0) EventContext<TestEvent> testEventContext) {
+        String testName = testEventContext.getEvent().getTestMethod().getName();
+
+        System.out.println("Starting test: " + testName);
```

Comment:
Can we use a logger here so that the liberty log shows which class is creating the log line?